lib/ansible/plugins/cliconf/vyos.py

```diff
@@ -109,6 +109,8 @@ def edit_config(self, candidate=None, commit=True, replace=None, comment=None):
                 self.discard_changes()
         else:
             self.send_command('exit')
```

Comment:
This executes both `exit` and `exit discard`. Is this expected or only `exit discard` should work?